cmd/ipam-node/main.go

```diff
@@ -13,7 +13,312 @@
 
 package main
 
+import (
+	"bytes"
+	"crypto/sha256"
+	b64 "encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"text/template"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"gopkg.in/k8snetworkplumbingwg/multus-cni.v4/pkg/cmdutils"
```

Comment:
can we borrow CopyFileAtomic and not rely on multus ? which will pull in all deps

Reply:
Sure will do